jscomp/runtime/caml_obj.ml

```diff
@@ -276,15 +283,15 @@ and aux_obj_compare (a: Caml_obj_extern.t) (b: Caml_obj_extern.t) =
     if not (O.hasOwnProperty b key) ||
        caml_compare (O.get_value a key) (O.get_value b key) > 0
     then
-      match !min_key with
-      | None -> min_key := Some key
+      match min_key.contents with
```

Comment:
Changes to this file can be undone?

Reply:
Actually undoing these changes breaks `test/bs_queue_test.js`.

Reply:
Hmm, I am a bit lost, is there an js diff without such change?

Reply:
Ah, I should update  bs_stdlib_mini accordingly, will do